src/app/pages/applications/applications.component.scss

```diff
@@ -171,6 +171,8 @@ mat-tab-group {
 
     .chart-button-row {
       bottom: 5px;
+	  width: 315px;
+	  height: 45px;
```

Comment:
How did you manage to circumvent our linter? :)
It should have ran during pre-commit git hooks. Please run manually via `yarn lint:fix`

Reply:
Deleting it, without staging the deletion. ;-)
That damn linter prevented me from pushing my previous PR's without any feedback what was wrong and added an absolute shit tone of translations into the PR that I was previously told to not-include in PR's.

This indent though >.< 
Will fix!